CONTRIBUTING.md

```diff
@@ -71,30 +104,19 @@ branch:
     mkdir tutorials/Spectral-Line-Fitting
 
 All files used by the tutorial -- e.g., example data files, the IPython
-notebook file itself -- should go in this directory. Now you can start writing
-the tutorial! Change directories into this new path and start up an
-IPython notebook server:
-
-    cd tutorials/Spectral-Line-Fitting
-    ipython notebook --matplotlib inline
-
-Create a new notebook file, and write away! (Following the Content Guidelines above.)
-Remember to place any extra files
-used by the tutorial in the directory with the notebook file, and place them
-under git version control.
+notebook file itself -- should go in this directory. 
 
 You will also need to edit the notebook file metadata.
```

Comment:
We actually don't use the metadata anymore -- we should consider just asking the contributors to add this info to the first cell in a pre-formatted block?

Reply:
deleted for now. when we know what we want them to do, we can add the instructions.